druid/src/window.rs

```diff
@@ -53,27 +46,14 @@ pub struct Window<T> {
     // delegate?
 }
 
-impl<T> PendingWindow<T> {
-    pub(crate) fn new(
-        root: impl Widget<T> + 'static,
-        title: LocalizedString<T>,
-        menu: Option<MenuDesc<T>>,
-    ) -> PendingWindow<T> {
-        PendingWindow {
-            root: WidgetPod::new(Box::new(root)),
-            title,
-            menu,
-        }
-    }
-
-    pub(crate) fn into_window(self, id: WindowId, handle: WindowHandle) -> Window<T> {
-        let PendingWindow { root, title, menu } = self;
+impl<T> Window<T> {
+    pub(crate) fn new(id: WindowId, handle: WindowHandle, desc: WindowDesc<T>) -> Window<T> {
         Window {
             id,
-            root,
-            size: Size::ZERO,
-            title,
-            menu,
+            root: WidgetPod::new(desc.root),
+            size: desc.size.unwrap_or(Size::ZERO),
```

Comment:
I think we want to keep this as `Size::ZERO`; the size in the window description is passed to the window builder, but is not the actual size of the window itself; we will get that back from the platform once the window connects.

Reply:
I changed it back to Size::ZERO